vault/resource_jwt_auth_backend_role.go

```diff
@@ -83,6 +83,12 @@ func jwtAuthBackendRoleResource() *schema.Resource {
 				Type: schema.TypeString,
 			},
 		},
+		"bound_claims_type": {
+			Type:        schema.TypeString,
+			Optional:    true,
+			Default:     "string",
```

Comment:
We should be able to strip this since it's not actually used below. That way it's clearer that Vault handles defaulting, and not the Terraform code.

Reply:
I just applied the requested change in a new commit.